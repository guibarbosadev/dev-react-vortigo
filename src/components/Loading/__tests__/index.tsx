import { render, screen } from '@testing-library/react';
import React from 'react';
import { Loading } from '..';

describe('<Loading />', () => {
    it('matches snapshot', () => {
        const { container } = render(<Loading />);

        expect(container).toMatchSnapshot();
    });

    it('has testid = "loading"', () => {
        render(<Loading />);
        expect(screen.getByTestId('loading')).toBeInTheDocument();
    });
});
