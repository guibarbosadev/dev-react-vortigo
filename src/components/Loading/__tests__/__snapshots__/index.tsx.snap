// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> matches snapshot 1`] = `
<div>
  <div
    data-testid="loading"
    id="loading"
  />
</div>
`;
